interface/wc_decision_icons.gfx

```diff
@@ -352,4 +352,8 @@ spriteTypes = {
 		name = "GFX_repay_money"
 		texturefile = "gfx\\interface\\decision_icons\\repay_action.tga"
 	}
+	spriteType = {
+		name = "GFX_create_blood_knights_order"
+		texturefile = "gfx\\interface\\decision_icons\\create_blood_knights_order.dds"
```

Comment:
The decision has no icon.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/81939569-91023580-9607-11ea-902d-28636a1a3bd6.png)

</details>

Reply:
But I put gfx pic in decision_icons and made a clear path for it. Hm